test/integration/kernel.spec.ts

```diff
@@ -8505,3 +8301,127 @@ describe('Kernel', () => {
 		});
 	});
 });
+
+// Create the relationships which are not already part of the kernel
+async function createRelationships() {
+	const relSpecs: any[] = [
+		{
+			fromVersionedType: 'user@1.0.0',
+			toVersionedType: 'org@1.0.0',
+			name: 'is member of',
+			inverseName: 'has member',
+			title: 'User',
+			inverseTitle: 'Organization',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'org@1.0.0',
+			name: 'is part of',
+			inverseName: 'has member',
+			title: 'Card',
+			inverseTitle: 'Organization',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'card@1.0.0',
+			name: 'is attached to',
+			inverseName: 'has attached element',
+			title: 'Attachment',
+			inverseTitle: 'Attachment',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'card@1.0.0',
+			name: 'is linked to',
+			inverseName: 'has link',
+			title: 'Link',
+			inverseTitle: 'Link',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'card@1.0.0',
+			name: 'is child of',
+			inverseName: 'has child', // It is defined like this in tests above
+			title: 'Card',
+			inverseTitle: 'Card',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'card@1.0.0',
+			name: 'is appended to',
+			inverseName: 'has appended element',
+			title: 'Card',
+			inverseTitle: 'Card',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'card@1.0.0',
+			name: 'is owned by',
+			inverseName: 'owns',
+			title: 'Card',
+			inverseTitle: 'Owner',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'card@1.0.0',
+			name: 'works at',
+			inverseName: 'has worker',
+			title: 'Worker',
+			inverseTitle: 'Workplace',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
+			toVersionedType: 'card@1.0.0',
+			name: 'believes in',
+			inverseName: 'is believed by',
+			title: 'Believer',
+			inverseTitle: 'Deity',
+		},
+		{
+			fromVersionedType: 'card@1.0.0',
```

Comment:
Am I mistaken that that types used in relationship from/to don't need to be versioned? I was under the impression a non-versioned type slug would be enough when it comes to defining relationships.

Reply:
We can use versioned or non-versioned types.
I used the versioned types because that's what is used when inserting the related `cards`, see for example l. 159
But using just the slug would also work, and if it's more clear then we can use just `card`.
I've updated the examples